### ISSUE TITLE
My oraxen resourcepack wont work these errors im getting in console

### ISSUE THREAD

**Issue by Jirkosaurcau** [oraxen/oraxen#565]
I dont know what to do with it can anybody help me?

ERRORS:

[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for explorer_tag does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for knight_tag does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for lord_tag does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for prince_tag does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for king_tag does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for hero_tag does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for god_tag does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for heart does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for logo does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_banner does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_recipe does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_items does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_items_overlay does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for required does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_1 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_2 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_4 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_8 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_16 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_32 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_64 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_128 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_256 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_512 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_1024 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_1 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_2 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_4 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_8 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_16 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_32 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_64 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_128 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_256 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_512 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_1024 does not exist. This will break all your glyphs.
[09:56:40] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:40] [Server thread/INFO]: Oraxen | Automatic upload of the resource pack is enabled, uploading...
[09:56:40] [Server thread/INFO]: Loaded 7 recipes
[09:56:40] [Server thread/INFO]: Loaded 7 recipes
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for farmer_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for explorer_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for knight_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for lord_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for prince_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for king_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for hero_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for god_tag does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for heart does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for logo does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_banner does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_recipe does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_items does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for menu_items_overlay does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for required does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_1 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_2 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_4 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_8 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_16 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_32 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_64 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_128 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_256 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_512 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for shift_1024 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_1 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_2 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_4 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_8 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_16 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_32 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_64 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_128 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_256 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_512 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.
[09:56:41] [Server thread/ERROR]: [null] [Oraxen] The texture specified for neg_shift_1024 does not exist. This will break all your glyphs.
[09:56:41] [Server thread/WARN]: [null] [Oraxen] It has been temporarily set to a placeholder image. You should edit this in the glyph config.

**What do i do?**

**Comment by Boy0000** [comment]
Did you compile the plugin?
its just saying the image for all these glyphs doesnt exist, plugin still works
TLDR u compiled and dont have default assets so default configs point to textures that doesnt exist
